src/analysis/normed/group/basic.lean

```diff
@@ -1020,6 +1020,14 @@ begin
   exact (tendsto_at_top_of_eventually_const this).cauchy_seq.add hv
 end
 
+lemma normed_group.tendsto_sub_nhds_zero_iff
```

Comment:
Can this be closer to `squeeze_zero_norm` and other `filter`-`norm` lemmas?

Reply:
No, because the proof uses the topological group instance which is proved just a few lines above.